Docs 10708 3.9 devkit important (#36)
mulesoft/docs-connector-devkit: modules/ROOT/pages/connector-attributes-and-operations.adoc

endif::[]
:keywords: devkit, attributes, operations, data, model, static, dynamic

include::partial$devkit-important.adoc[]

> Once you have your authentication working, you can start adding functionality to your connector. Adding functionality requires the following steps:

Reviewer: ```suggestion
After you have authentication working, you can start adding functionality to your connector by:
```
